apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html

```diff
@@ -53,8 +53,16 @@ <h3 class="modal-title pull-left">{{ localeData?.approve }}</h3>
         </div>
 
         <div>
-            <button class="btn btn-warning" (click)="approveEntry.emit(false)">{{ commonLocaleData?.app_no }}</button>
-            <button class="btn btn-success" (click)="approveEntry.emit(true)" [ladda]="approveEntryRequestInProcess">
+            <button mat-raised-button color="accent" (click)="approveEntry.emit(false)">
+                {{ commonLocaleData?.app_no }}
+            </button>
+            <button
+                mat-raised-button
+                color="primary"
```

Comment:
remove the color and use class name properly. @ravinderwalkover

Reply:
Done